project/Dependencies.scala

```diff
@@ -2,14 +2,14 @@ import sbt._
 
 object Dependencies {
 
-  private val gatlingVersion = "2.2.5"
-
-  object Compile {
-    val typesafeConfig = "com.typesafe" % "config" % "1.3.1"
-    val performanceTestRunner = "uk.gov.hmrc" %% "performance-test-runner" % "3.1.0"
-    val gatlingTestFramework = "io.gatling" % "gatling-test-framework" % gatlingVersion
-    val gatlingHighCharts = "io.gatling.highcharts" % "gatling-charts-highcharts" % gatlingVersion
-  }
+  private val gatlingVersion = "2.3.1"
+
+  val test = Seq(
+    "com.typesafe" % "config" % "1.3.1" % Test,
+    "uk.gov.hmrc" %% "performance-test-runner" % "3.7.0" % Test,
+    "io.gatling" % "gatling-test-framework" % gatlingVersion % Test,
+    "io.gatling.highcharts" % "gatling-charts-highcharts" % gatlingVersion % Test
+  )
 }
 
 
```

Comment:
As a side note, is a Dependencies.scala the preferred pattern to defining the dependencies in the build.sbt?  

Personally I normally find it easier to have it all in one place, but if that's the HMRC pattern so be it.